tests/test_hvcs.py

```diff
@@ -202,11 +201,60 @@ def test_get_domain_should_have_expected_domain(
                 "CI_SERVER_HOST": ci_server_host,
             },
         ):
-
             assert get_hvcs().domain() == expected_domain
             assert get_hvcs().api_url() == expected_api_url
 
 
+@responses.activate
+@pytest.mark.parametrize(
+    "hvcs,url,prerelease",
+    [
+        ("github", "https://api.github.com/repos/relekang/rmoq/releases", True),
```

Comment:
I think it'd be cleaner passing the class in as `hvcs` instead of a string, and if you do two `parametrize` decorators you can get the same effect with less params per test; i.e.

```python
@pytest.mark.parametrize(
    "hvcs, url",
    [
        (Github, "https://api.github.com/repos/relekang/rmoq/releases"),
        (Gitea, "https://gitea.com/api/v1/repos/gitea/tea/releases")
    ]
)
@pytest.mark.parametrize("prerelease", (True, False))
def test_should_create_prerelease_if_asked_for(hvcs, url, prerelease):
    ...
```

That (+ the netrc line) might have cut down on the `if`s in the test itself - but I don't see anything wrong with what you've done here :+1: